### PR TITLE
Integrate maven

### PR DESCRIPTION
**Pull request by jcorteza**
https://confluence.dev.lithium.com/display/CD/Lab%3A+Dependency+Management
Not sure if I did step 4 properly, but it the app works when you run 
java -jar /TwitterApp/target/TwitterApp-1.0-SNAPSHOT.jar check_feed

### CODE REVIEW

**Review by kev-chan — CHANGES_REQUESTED** [review]
a few things:
- have the pom.xml file in the root directory so it can be compiled there vs in a subfolder
- update readme to have the proper file name vs <app name>
- i'll look some more afterwards

**Comment by kev-chan** [comment]
the src file should be in root directory as well
- don't check in the class files
- don't check in the target directory

**Comment by jcorteza** [comment]
I also added **/classes and **/target to my .gitignore

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
```
Comment (kev-chan) [inline pom.xml]: have the dependency versions in properties

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+           <groupId>org.twitter4j</groupId>
+           <artifactId>twitter4j-core</artifactId>
+           <version>[4.0,)</version>
```
Comment (kev-chan) [inline pom.xml]: same as above

**Review by kev-chan — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (jcorteza) [reply pom.xml]: Done.

File: pom.xml (hunk 2 above)
Comment (jcorteza) [reply pom.xml]: done.

**Review by kev-chan — CHANGES_REQUESTED** [review]
add error checking for tweet content when posting

**Pull request merged**